### DISCUSSION TITLE
Umbraco Forms - Send Email Workflow

### DISCUSSION THREAD

**Discussion by Ric-Carey** [umbraco/Umbraco-CMS#12820] [Features and ideas]
Hey,

Would it be possible to add a config setting to set the path to the default email template which is used in the workflow which sends an email. Clients regularly get really annoyed when the emails are branded in the Umbraco Forms default one. 

Also, on the default email template, no offence to the people in the photo but could that be changed to something more generic Umbraco than a photo of female developers. Most clients don’t have a clue who these people are, and makes it look like its a photo of friends that the developer working on the project has put in there on purpose. 

In Umb 8 and below, we used to just be able to completely overwrite this email template and avoid confusion for editors, but this is no longer the case anymore with .net Core reverting to the default. I would suggest that this increases the need to make this email template as generic as possible and think of the UX from an editor receiving that email's POV. 

Cheers
Ric

**Comment by nul800sebastiaan** [comment]
Probably better to send a feature request to the Forms issue tracker: https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
